lottie-react-native.podspec

```diff
@@ -1,26 +1,21 @@
 require 'json'
 
-package = JSON.parse(File.read(File.join(__dir__, 'package.json')))
+package = JSON.parse(File.read(File.join(__dir__, "package.json")))
 
 Pod::Spec.new do |s|
-  s.name         = "lottie-react-native"
+  s.name         = package["name"]
   s.version      = package['version']
-  s.summary      = "Lottie component for React Native (iOS and Android)"
-
-  s.authors      = { "intelligibabble" => "leland.m.richardson@gmail.com" }
-  s.homepage     = "https://github.com/airbnb/lottie-react-native#readme"
-  s.license      = package['license']
+  s.summary      = package["description"]
+  s.homepage     = package["homepage"]
+  s.license      = { :type => "Apache-2.0", :file => "LICENSE" }
+  s.author      = { "author" => "leland.m.richardson@gmail.com" }
   s.ios.deployment_target = '11.0'
   s.osx.deployment_target = '10.10'
   s.tvos.deployment_target = '11.0'
-
   s.source       = { :git => "https://github.com/react-community/lottie-react-native.git", :tag => "v#{s.version}" }
```

Comment:
This needs to be updated to: `https://github.com/lottie-react-native/lottie-react-native.git`